apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -32,6 +32,20 @@
         Context :: #{}
                    ) -> {Status :: cowboy:http_status(), Headers :: list(), Body :: map()}.
 
+handle_request('PostKeyBlock', #{'KeyBlock' := Data}, _Context) ->
+    case aec_headers:deserialize_from_client(key, Data) of
+        {ok, Header} ->
+            KeyBlock = aec_blocks:new_key_from_header(Header),
+            case aec_conductor:post_block(KeyBlock) of
```

Comment:
I see - based on code inspection - that the conductor clears and updates the key block candidate (pending block) after the mined block is posted. Ok.